_posts/2024-10-15-zio-full-stack-webapp.md

```diff
@@ -609,7 +609,7 @@ Everything is described as ZIO effects. `URIO[PersonService, PersonController]`
 
 > **URIO** is a type alias for a ZIO that cannot fail:
 >
->  `type URIO[R, A] = ZIO[R, Nothing, A]`
+>  type **URIO**[R, A] = ZIO[R, **Nothing**, A]
```

Comment:
I intentionally set these as <pre> blocks but if you think the bolding is more important, ok for me